svgdigitizer/entrypoint.py

```diff
@@ -504,6 +504,15 @@ def _create_linked_svg(svg, png, svg_template):
     r"""
     Write an SVG to `svg` that shows `png` as a linked image.
 
+    This is a helper method for :meth:`paginate`.
+    """
+    _create_svg(svg, png, svg_template, True)
+
+
+def _create_svg(svg, png, svg_template, linked):
```

Comment:
```suggestion
def _create_svg(svg, image, svg_template, linked):
```

Reply:
Apparently, below, you also allow already jpeg. So maybe this argument should be more generic

Reply:
I think linked means that the SVG links to a PNG from source and embed refers to actually including the PNG in the SVG

Reply:
sorry now I get the reasoning. I was looking from the embed perspective...